app/views/map/_peopleLeaflet.html.erb

```diff
@@ -7,7 +7,9 @@
 	<% if defined? people %>
 		<p><i><small>
```

Comment:
@jywarren any idea why we're using `<i>` tags for italicizing text and not CSS like everyone else? IIRC, the `<i>` tag must be deprecated.

Reply:
thanks for the change requests @ryzokuken just wondering how i go about removing the gemfile.lock from the pull request

Reply:
^ for some CSS it'd be a lot more verbose, and we can cue a specific style by setting CSS for the very compact `<i>` -- it's not completely deprecated as FontAwesome uses it, for example -- sometimes it's nice to have this without taking up too much space!